components/CDT/wmtk/components/CDT/CDT.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <wmtk/Mesh.hpp>
+
+namespace wmtk::components {
+
+std::shared_ptr<Mesh>
+CDT(const std::shared_ptr<Mesh>& input, const bool inner_only, const bool rational_output);
```

Comment:
I prefer to not pass pointers, also should be TriMesh, no?

Reply:
yeah I will change to reference.